src/app/settings/settings.html

```diff
@@ -84,6 +84,17 @@ <h1 ng-i18next="Settings.Settings"></h1>
           <input type="checkbox" id="allowIdPostToDtr" ng-model="vm.settings.allowIdPostToDtr"/>
         </td>
       </tr>
+      <tr ng-if="vm.settings.destinyVersion === 2 && (vm.settings.showReviews || vm.settings.allowIdPostToDtr)">
+        <td>
+          <label for="reviewPlatformSelection" ng-i18next="Settings.PlatformSelection"></label>
+        </td>
+        <td>
+          <select id="reviewPlatformSelection"
+            ng-model="vm.settings.platformSelection"
+            ng-options="code as name for (code, name) in vm.platformOptions"
+            ng-change="vm.platformChanged()"></select>
```

Comment:
Same with the other "platforms".

Reply:
The plural above I follow, but I don’t understand this.

Reply:
`vm.settings.reviewsPlatformSelection`, `vm.reviewsPlatformOptions`, `vm.reviewsPlatformChanged()`.